cms/envs/common.py

```diff
@@ -813,13 +827,25 @@
     'common/js/vendor/require.js'
 ]
 
+capa_js = (
+    rooted_glob(COMMON_ROOT / 'static', 'common/js/javascript_loader.js') +
```

Comment:
It looks like the `rooted_glob()` function expands glob patterns like `*` and `?`. Since you don't have any wildcard characters, can't you simply use the path name instead?

Reply:
Sure.